browser-test/src/application_review.test.ts

```diff
@@ -280,7 +280,7 @@ test.describe('Program admin review of submitted applications', () => {
       await validateScreenshot(page, 'reporting-page')
     }
 
-    await page.click(`text=${programName.replaceAll(' ', '-').toLowerCase()}`)
+    await page.getByRole('link', { name: programName })
```

Comment:
```suggestion
    await page.getByRole('link', { name: programName }).click()
```

Sorry. Missed adding the click for you.